README.md

```diff
@@ -39,6 +39,89 @@ bundle install
 rake -T
 ```
 
+## Dockerfile
+
+### Build image
+```
+docker build -t <image_name> .
+```
+### Run tests in Docker
+
+Driver which is used by default is **headless chrome**
+
+Use **docker run** command to create and run container.
+
+- In order to run container by default:
+```
+docker run -d --name <container_name> <image_name>
+```
+
+- To run container using firefox headless browser need to override environment variable:
+**SEXY_SETTINGS**
+```
+docker run -e "SEXY_SETTINGS=driver=headless_firefox" -d --name <container_name> <image_name>
+```  
+
+- Connect to created container using this command:
+```
+docker exec -it <container_name/ID> /bin/bash
+```
+
+- Run tests with the next command using container's terminal:
+```
+bundle exec rake
```

Comment:
I believe we have to remove SEXY_SETTINGS from Dockerfile and use it here like:

```
SEXY_SETTINGS=xxx bundle exec rake
```

Reply:
How about such kind of commands? 
SEXY_SETTINGS="driver=headless_chrome; $CHROME_ARGS" bundle exec rake
SEXY_SETTINGS="driver=headless_firefox" bundle exec rake